source/SpinalHDL/Semantic/assignments.rst

```diff
@@ -118,3 +118,51 @@ Combinatorial loops
 
 SpinalHDL checks that there are no combinatorial loops (latches) in your design.
 If one is detected, it raises an error and SpinalHDL will print the path of the loop.
+
+CombInit
+--------
+
+The special ``CombInit`` method can be used to "clone" a combinatorial value for latter modification.
```

Comment:
I'd try to distinguish this from `cloneOf` (and a small technical change since CombInit is not really a method) - a wording could be:
```suggestion
``CombInit`` can be used to copy a signal and its current combinatorial assignments. The main use-case is to be able to overwrite the copied later, without impacting the original signal.
```